modules/renter/contractor/contracts.go

```diff
@@ -318,7 +318,7 @@ func (c *Contractor) threadedContractMaintenance() {
 	var endHeight types.BlockHeight
 	var fundsAvailable types.Currency
 	var renewSet []renewal
-	refreshSet := make(map[types.FileContractID]struct{})
+	refreshSet := make(map[string]struct{})
```

Comment:
is there a reason that you changed this type from a `types.FileContractID` to a `string`?

Reply:
lol I just realized that we never used this set anyway. Seems like we can delete that.
The reason for the string was the change to a pubKey instead of a contract id. Go can't use byte slices as a mapkey.